python_ta/__init__.py

```diff
@@ -531,3 +581,61 @@ def doc(msg_id: str) -> None:
     msg_url = HELP_URL + "#" + msg_id.lower()
     print("Opening {} in a browser.".format(msg_url))
     webbrowser.open(msg_url)
+
+
+def _watch_files(
```

Comment:
This function should print (via logging) a message saying "PythonTA is monitoring your files for changes and will re-check after every save"